src/main.py

```diff
@@ -33,12 +34,10 @@ def main():
     turn = 1
     cond = 20

```

Comment:
now we have another size `480x640`, in that case, this value must be lower (10 for example)